listenstore/listenstore/listenstore.py

```diff
@@ -31,7 +31,6 @@ def __init__(self, conf):
     def max_id(self):
         return int(self.MAX_FUTURE_SECONDS + calendar.timegm(time.gmtime()))
 
-
```

Comment:
Sorry!
I always try to clean the code as much as possible. I just can't let these redundant lines pass into the code base. I always try to clean the code gradually, so with every PR i fix a lot of pylint issues and try to improve the logic.
If this needs to be removed then i'll remove it.